cms/envs/aws.py

```diff
@@ -207,6 +207,7 @@
 SERVER_EMAIL = ENV_TOKENS.get('SERVER_EMAIL', SERVER_EMAIL)
 MKTG_URLS = ENV_TOKENS.get('MKTG_URLS', MKTG_URLS)
 MKTG_URL_LINK_MAP.update(ENV_TOKENS.get('MKTG_URL_LINK_MAP', {}))
+PROGS_URLS = ENV_TOKENS.get('PROGS_URLS', PROGS_URLS)
```

Comment:
Please move all changes to be under the `################ Edraak Apps ###############` section.

Same goes for all environment files.

Reply:
Done